gluster/driver/tools_test.go

```diff
@@ -18,9 +18,13 @@ func TestIsValidURI(t *testing.T) {
 		{"test,volume", false},
 		{"test,test2:volume", true},
 		{"192.168.1.1:volume", true},
-		{"192.168.1.:volume", false},
+		{"192.168.1.:volume", true},
```

Comment:
From previous comments this should fail

Reply:
Feel free to update the PR. However, I still think making it loose will prevent future cases when some people have atypical but valid host names or volume names. Is there a way of extracting the error message so it can be displayed rather than status code is 1?

Reply:
Hopefully we'd get a release I can try so I don't have to spend the effort of creating extra volumes.